### PR TITLE
Added Pylons for AH6

### PR DESCRIPTION
**Pull request by Rios92**
- Added CUP/Vanilla Pylons to the AH6 to the outward hardpoints.
- Added Dynamic Loadouts file for it to be shown in ACE Pylon cofiguration.
- tft_heli was unpacked from the TFT_Private, then repacked on 7/26/2020.

### CODE REVIEW

**Comment by ampersand38** [comment]
> unpacked ... repacked
You literally have the source code here. Why are you unpacking it from the pbo?

`dynamicLoadouts.hpp` is not used in a `#include` anywhere. That code is literally not the game at all.

**Comment by Rios92** [comment]
The source code of the pylons were not there. I opened up tft_heli via pbo manager to extract it the config file to add in `
transportpyloncomponent
` for the `RHS_MELB_AH6` class.

File: tft_heli/config.cpp [hunk 1]
```diff
@@ -3847,6 +3850,38 @@ class CfgVehicles {
...
+					};
+					class pylon3: pylon2
+					{
+						hitpoint="HitPylon3";
+					};
+					class pylon4: pylon1
+					{
+						hardpoints[]+= {"RHS_HP_MELB", "RHS_HP_MELB_R", "DAR", "DAGR", "B_SHRIEKER", "CUP_NATO_HELO_SMALL", "CUP_NATO_HELO_AH6", "CUP_NATO_HELO_AH6_RIGHT"};
+						hitpoint="HitPylon4";
+					};
```
Comment (ampersand38) [inline tft_heli/config.cpp:3880]: Why do you need to set this stuff?

File: tft_heli/config.cpp [hunk 2]
```diff
@@ -1,4 +1,5 @@
-﻿class CfgPatches {
+#include "BIS_AddonInfo.hpp"
+class CfgPatches {
```
Comment (ampersand38) [inline tft_heli/config.cpp:2]: Remove these changes

**Pull request merged**